lib/ffmpeg/encoding_options.rb

```diff
@@ -126,7 +130,16 @@ def convert_seek_time(value)
     end
 
     def convert_screenshot(value)
-      value ? "-vframes 1 -f image2" : ""
+      vframes = '-vframes 1 ' unless self[:vframes]
+      value ? "#{vframes}-f image2" : ""
```

Comment:
Previously, the code forced `vframes` to 1 when using the `screenshot` method. How do you like this impl? The idea is to keep the old behavior unless the developer specified `vframes`.